### ISSUE TITLE
Consider changing from creating Jobs to Pods

### ISSUE THREAD

**Issue by dleehr** [Duke-GCB/calrissian#22]
Calrissian creates a Kubernetes `Job` for each CWL CommandLineTool job to run. The [KubernetesJobBuilder.build](https://github.com/Duke-GCB/calrissian/blob/master/calrissian/job.py#L145) method builds the single-container definition.

However, to determine the CWL CommandLineTool's exit code, #21 makes a change to inspect each k8s Job's first Pod, and includes some code to check that the job had 1 pod and the pod had 1 container.

After implementing this, I'm not sure the k8s Job is what we should be submitting. Jobs are controllers in k8s and can have multiple replicas or cron behaviors. Additionally, k8s will restart a Job's pod(s) on a different node if it doesn't complete once. I don't think we need that layer of management, since calrissian is already watching/managing those details.

tl;dr: can we update calrissian to submit Pods instead of Jobs?
